app/assets/javascripts/controllers/container_deployment/deploy-provider-controller.js

```diff
@@ -38,6 +38,7 @@ miqHttpInject(angular.module('miq.containers.providersModule', ['ui.bootstrap',
 
       $scope.data.existingProviders = $scope.deploymentData.providers;
       $scope.data.newVmProviders = $scope.deploymentData.provision;
+      $scope.originAvailable = $.inArray("openshiftOrigin", $scope.deploymentData.deployment_types) > -1;
```

Comment:
This should probably be `$scope.originAvailable = $scope.deploymentData.deployment_types.includes("openshiftOrigin");`, otherwise, LGTM